thunderfish/sortfishtype.py

```diff
@@ -0,0 +1,177 @@
+import numpy as np
+import peakdetection as pkd
```

Comment:
import standard libraries first, then derived stuff, and finally local modules:
numpy
pyplot
peakdetection